hail/src/main/scala/is/hail/annotations/StagedRegionValueBuilder.scala

```diff
@@ -79,16 +79,16 @@ object StagedRegionValueBuilder {
       case t =>
         Code(
           offset := region.allocate(t.alignment, t.byteSize),
-          deepCopy(fb, region, t, value, offset))
+          deepCopy(fb, region, t, Region.loadIRIntermediate(t)(value), offset))
```

Comment:
this should technically be `getIRIntermediate`, although the cases in which the two diverge are already covered by the non-default cases.